### PR TITLE
Air Gap AlmaLinux VM

### PR DESCRIPTION
**Pull request by Param-29**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Comment by Param-29** [comment]
```sh
    # [vagrant@almalinux-9 ~]$ sudo iptables-save
    # Generated by iptables-save v1.8.8 (nf_tables) on Sat Sep 16 12:33:51 2023
    # *filter
    # :INPUT DROP [4:304]
    # :FORWARD DROP [0:0]
    # :OUTPUT ACCEPT [81:9418]
    # -A INPUT -p tcp -m tcp --dport 22 -j ACCEPT
    # COMMIT
    # # Completed on Sat Sep 16 12:33:51 2023
    
```

Settings output above

File: packages/grid/vagrant/fedora_38/x86_64/Vagrantfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Vagrant.configure("2") do |config|
-    config.vm.box = "bento/fedora-38-x86_64"
-    config.vm.hostname = "fedora-38"
+    config.vm.box = "bento/fedora-38"
```
Comment (Param-29) [inline packages/grid/vagrant/fedora_38/x86_64/Vagrantfile:2]: > https://app.vagrantup.com/bento/boxes/fedora-38
update to the name

File: packages/grid/vagrant/almalinux_9/x86_64/Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    iptables-save > rules.v4.old
+    iptables -F
+
+    # Set the default policies to DROP
+    iptables -P INPUT DROP
+    iptables -P FORWARD DROP
+    iptables -P OUTPUT ACCEPT
+
+    # Allow incoming SSH traffic
+    iptables -A INPUT -p tcp --dport 22 -j ACCEPT
```
Comment (Param-29) [inline packages/grid/vagrant/almalinux_9/x86_64/Vagrantfile:29]: SSH allowed via this

**Comment by Param-29** [comment]
```
[vagrant@almalinux-9 ~]$ ls
rules.v4.new  rules.v4.old
[vagrant@almalinux-9 ~]$ cat rules.v4.new 
# Generated by iptables-save v1.8.8 (nf_tables) on Tue Sep 19 09:20:21 2023
*filter
:INPUT DROP [0:0]
:FORWARD DROP [0:0]
:OUTPUT ACCEPT [0:0]
-A INPUT -p tcp -m tcp --dport 22 -j ACCEPT
COMMIT
# Completed on Tue Sep 19 09:20:21 2023
[vagrant@almalinux-9 ~]$ curl www.google.com
curl: (6) Could not resolve host: www.google.com
[vagrant@almalinux-9 ~]$ curl https://github.com/OpenMined/PySyft
curl: (6) Could not resolve host: github.com
[vagrant@almalinux-9 ~]$ 

```

**Review by rasswanth-s — APPROVED** [review]
Well Done @Param-29

**Comment by madhavajay** [comment]
This old PR contains the github action example to run VirtualBox on GitHub MacOS Runners. https://github.com/OpenMined/PySyft/pull/7579

**Comment by rasswanth-s** [comment]
I think , we could merge this PR to keep the dev velocity, and add tests for vagrant in subsequent PR's due to our current upcoming milestones.

I have verified the initial vagrant setup in my ARM machine manually.

**Review by rasswanth-s — APPROVED** [review]
Superb Work @Param-29

**Pull request merged**